src/IndexingMsg.chpl

```diff
@@ -607,32 +607,44 @@ module IndexingMsg
             when (DType.BigInt, DType.BigInt) {
                 var e = toSymEntry(gEnt,bigint);
                 var val = valueArg.getBigIntValue();
+                var max_size = 1:bigint;
```

Comment:
i think let's declare this `var` within the `if` since it's only ever used `if e.max_bits != -1`, so there's no point declaring it if not

Reply:
Updated